components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -4296,6 +4296,36 @@ def searchProcessing():
             search.close()
         return rv
 
+    def getThumbnailSet(self, image_ids, max_size=64):
```

Comment:
webgateway was switched to use size 96 by default https://github.com/openmicroscopy/openmicroscopy/pull/4687 which is also what this PR uses.
So I guess this PR is consistent with existing behaviour. The only question is *if* we're moving Blitz Gateway to use size 96 by default, do we want to use that here? If thumbnails of size 96 are cached, then using size 64 will be slower. cc @chris-allan